src/panoptoindexconnector/implementations/microsoft_graph_implementation.py

```diff
@@ -85,6 +88,16 @@ def push_to_target(target_content, config):
 
     if response.status_code == 200:
         LOG.info(f"Content ({content_id}) has been pushed to target!")
+    # If request is forbidden
+    elif response.status_code == 403:
+        error = response.json()["error"]
+
+        if error and error.get("innerError"):
+            innerError = error.get("innerError")
+            if innerError.get('code') == "TenantQuotaExceeded":
+                raise CustomExceptions.QuotaLimitExceededError(innerError.get("message"))
+
+        response.raise_for_status()
```

Comment:
Is this line a residual of development? Also, isn't it true that your new code now loses a log for the case that the response is 403 but not from quota issue? Before your change, it went to else clause and it has a log line.

Reply:
You are right. Added missing log. Thanks for catching!